R/ddl.R

```diff
@@ -0,0 +1,223 @@
+#' DDL object
+#'
+#' Object to execute custom DDL code in the shiny session
+#'
+#' @param expr (`expression`)\cr
+#'  Syntatically valid R code to be executed in the shiny session.
+#'  shouldn't be specified when `code` is specified.
+#'
+#' @param code (`character`, `language`)\cr
+#'  Object containing code to be evaluated to load data. Shouldn't be specified when `expr`
```

Comment:
it's more visible is you split to `verb NOT`
```suggestion
#'  Object containing code to be evaluated to load data. Should not be specified when `expr`
```